README.md

```diff
@@ -29,6 +29,12 @@ This will install the CLI into the `/usr/local/bin` directory.
 
 If you do not have write permissions to `/usr/local/bin`, you may need to run the above command with `sudo`.
 
+Optionally, you can install in a arbitrary directory by using `DESTDIR` environment variable.
+
+```
+export DESTDIR=/path/to/install/ && bash -c "$(curl -fSl https://raw.githubusercontent.com/CircleCI-Public/circleci-cli/master/install.sh)"
```

Comment:
Does `DESTDIR=/path/to/install bash -c "$(...)"` not do the same thing?

I also wonder if we should make this a command-line flag instead of an environment variable.

Reply:
See https://github.com/CircleCI-Public/circleci-cli/issues/153 for more context. I'm happy with an env var.

Reply:
`DESTDIR=/path/to/install bash -c "$(...)"` should do it. Is `DESTDIR` a clear enough word?

There seems to be some prior art for this word [in `automake`]((https://www.gnu.org/software/automake/manual/html_node/DESTDIR.html).

Reply:
Thanks for reviewing for both of two. 

@zzak I personally think it's enough to use env var, because we can do same thing by both way, env var and command-line flag, so I took easier way. (of courese, i think there's some trade off)
But, i don't mind to use command-line flag.
if you have some good reason to use command-line flag, give me some instruction.

@marcomorain 

I got it. I didn't know much about `automake`. 

I first tryed to use `DEST` actually, but `DEST` has been already used in install.sh (which is fully qualified path). My opinion is like below.

- When to use `DEST`
  - I think it's better to change `DEST` in install.sh to other name, like `EXEC_PATH`.
  - Other
- Other options: not to use DEST*. Some ideas are below.
  - DSTDIR
  - DIR
  - TRAGETDIR
  - EXECDIR